Sources/KlaviyoCore/Networking/PrivateMethods.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  File.swift
+//
+//
+//  Created by Ajay Subramanya on 8/29/24.
+//
+
+import Foundation
+
+@_spi(KlaviyoPrivate)
+/// Used to override SDK defailts for INTERNAL USE ONLY
+/// - Parameter url: The  URL to use for Klaviyo client APIs, This is used internally to test the SDK against different backends, DO NOT use this in your apps.
+/// - Parameter name: The name of the SDK, defaults to swift but react native will pass it's own name. DO NOT override this in your apps as our backend will not accept unsupported values here and your network requests will fail.
+/// - Parameter version: The version of the swift SDK default to hard coded values here but react native will pass it's own values here. DO NOT override this in your apps.
```

Comment:
```suggestion
/// Used to override SDK defailts for INTERNAL USE ONLY
/// - Parameter url: The  URL to use for Klaviyo client APIs, This is used internally to test the SDK against different backends, DO NOT use this in your apps.
/// - Parameter name: The name of the SDK, defaults to swift but react native will pass it's own name. DO NOT override this in your apps as our backend will not accept unsupported values here and your network requests will fail.
/// - Parameter version: The version of the swift SDK default to hard coded values here but react native will pass it's own values here. DO NOT override this in your apps.
@_spi(KlaviyoPrivate)
```
nit: I was playing around and it looks like the @_spi tag needs to go below the documentation in order for the documentation to show up properly. See these screenshots:

<table>
  <tr>
    <td>@_spi tag above documentation</td>
     <td>@_spi tag below documentation</td>
  </tr>
  <tr>
    <td><img src="https://github.com/user-attachments/assets/ee2fc12d-833d-4b00-803d-1a543c661f60" width=270></td>
    <td><img src="https://github.com/user-attachments/assets/c575a83a-5644-4a1a-a802-c3f0df9252ef" width=270></td>
  </tr>
 </table>

Reply:
wow great find!

Reply:
Weird, I used the xcode shortcut (cmd + opt + /) to generate comments and it pushed the @_spi above the comments. Thanks for looking into this locally. I'll update it.